mesheryctl/internal/cli/root/mesh/mesh.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Anirudh Jain
```

Comment:
Currently, the vast majority of our files do not include a license header. However, some do and including one is fine. Layer5 projects use Apache v2 licensing. Copyright is not retained by the individual, but assigned to the collective whole. We'll need to either remove this header or include this standard license header instead:

```
// Copyright 2020 Layer5 Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```